nixos/tests/ceph.nix

```diff
@@ -1,4 +1,4 @@
-import ./make-test.nix ({pkgs, ...}: rec {
+import ./make-test.nix ({pkgs, lib, ...}: rec {
   name = "All-in-one-basic-ceph-cluster";
   meta = with pkgs.stdenv.lib.maintainers; {
     maintainers = [ lejonet ];
```

Comment:
Would you like to be a maintainer?

Reply:
I guess I should add myself to maintainers. So yes, I want to be a maintainer.